content/docs/use-cases/shared-development-server.md

```diff
@@ -3,102 +3,110 @@
+shared <abbr>cache</abbr> location for everyone's projects:
 
 ```dvc
-$ mkdir -p /path/to/dvc-cache
+$ mkdir -p /home/shared/dvc-cache
 ```
 
-You will have to make sure that the directory has proper permissions setup, so
-that all your colleagues can read and write to it, and can access cache files
-written by others. The most straightforward way to do this is to make sure that
-everyone's users are members of the same group, and that your shared cache
-directory is owned by this group, with the aforementioned permissions.
+> The `/home/shared` directory used as example above is typical in Linux
+> distributions.
 
-## Transfer existing cache (Optional)
+Make sure that the directory has proper permissions, so that all your colleagues
+can write to it, and can read cached files written by others. The most
+straightforward way to do this is to make all users members of the same group,
+and have the shared cache directory owned by that group.
 
-This step is optional. You can skip it if you are setting up a new DVC project
-whose cache directory is not stored in the default location, `.dvc/cache`. If
-you did work on your project with DVC previously and you wish to transfer your
-cache to the shared cache directory (external to your workspace), you will need
-to simply move it from an old cache location to the new one:
+## Transfer existing cache (optional)
+
+You can skip this part if you are setting up a new DVC project where the local
+<abbr>cache directory</abbr> (`.dvc/cache` by default), hasn't been used.
+
+If you did work on the <abbr>DVC projects</abbr> previously and wish to transfer
+its existing cache to the shared cache directory, you will simply need to move
+its contents from the old location to the new one:
 
 ```dvc
-$ mv .dvc/cache/* /path/to/dvc-cache
+$ mv .dvc/cache/* /home/shared/dvc-cache
 ```
 
-Now you need to ensure that cache files/directories have appropriate
-permissions, so that they could be accessed by your colleagues that are members
-of the same group:
+Now, ensure that the cached directories and files have appropriate permissions,
+so that they can be accessed by your colleagues (assuming their users are
+members of the same group):
 
 ```dvc
-$ sudo find /path/to/dvc-cache -type f -exec chmod 0664 {} \;
-$ sudo find /path/to/dvc-cache -type d -exec chmod 0775 {} \;
-$ sudo chown -R myuser:ourgroup /path/to/dvc-cache/
+$ sudo find /home/shared/dvc-cache -type d -exec chmod 0775 {} \;
+$ sudo find /home/shared/dvc-cache -type f -exec chmod 0664 {} \;
+$ sudo chown -R myuser:ourgroup /home/shared/dvc-cache/
 ```
 
-## Configure shared cache
+## Configure the external shared cache
 
-Tell DVC to use the directory we've set up above as an shared cache location by
-running:
+Tell DVC to use the directory we've set up above as the <abbr>cache</abbr> for
+your <abbr>project</abbr>:
 
 ```dvc
-$ dvc config cache.dir /path/to/dvc-cache
+$ dvc config cache.dir /home/shared/dvc-cache
 ```
```

Comment:
Why do we have all these exact and reproducible command guides in a use case? This section seems more like a user guide to me.

Reply:
yes, I agree ... most of it should be part of the `shared cache` article in the UG. And use case should be probably more generic "Optimizing data management" which can cover this one (multiple people working on a single box), k8s scenario - multiple machines + NAS, etc. We are long overdue on this. There were contributors that made an attempt, but didn't get it right.

Reply:
OK. I see we have #429 for something like this already. And #986 is also related.  Should we try to consolidate them into a single issue and include the comments from this review?

Reply:
so, these two mostly about improving the existing case - but they are relevant, up to you to consolidate or not ... there should be tickets related to this from other angles - shared cache on NAS/NFS for example

Reply:
Some other related links:

https://discuss.dvc.org/t/setup-dvc-to-work-with-shared-data-on-nas-server/180/4
https://github.com/iterative/dvc.org/pull/732
https://github.com/iterative/dvc.org/pull/455
https://github.com/iterative/dvc.org/issues/563

and others ... someone should a first step systematize and suggest the way to structure this properly